CHANGES.rst

```diff
@@ -228,6 +228,11 @@ API Changes
   - Moved ``units.cgs.emu`` to ``units.deprecated.emu`` due to ambiguous
     definition of "emu". [#4918, #5906]
 
+  - ``solLum``,``solMass``, ``jupiterMass``, and ``earthMass``, as well as
+    ``solRad``, ``jupiterRad``, and ``earthRad`` no longer have their prefixed
+    units included in the standard units.  They can now be found in
+    ``units.deprecated``. [#5661]
```

Comment:
I'm not sure we can eventually remove these deprecated units, but I would still phrase this a bit more discouragingly, maybe as "If needed, they can still ..."